src/utils.jl

```diff
@@ -158,13 +23,21 @@ function fullvector(M::MPS; reverse::Bool = true)
 end
 
 """
-    PastaQ.fullmatrix(M::MPO; reverse::Bool = true)
-    PastaQ.fullmatrix(L::LPDO; reverse::Bool = true)
+    array(M::MPO; reverse::Bool = true)
+    array(L::LPDO; reverse::Bool = true)
 
-Extract the full matrix from an MPO or LPDO, returning a Julia Matrix.
+Generate the full dense matrix from an MPO or LPDO.
 """
-function fullmatrix(M::MPO; reverse::Bool = true)
-  s = firstsiteinds(M; plev = 0)
+function ITensors.array(M::MPO; reverse::Bool = true)
```

Comment:
Defining functions like this is not great (this is called type piracy, where you overload a function that is defined in another package with a type that is also defined in that package). We could consider this but we would need to make this function more general (i.e. not specific to the "Input" and "Output" tags). 

Could we name this something else, like `choi_to_array`?

Reply:
That function works for both Choi and regular MPOs

Reply:
Maybe we can change the name to `toarray` to avoid this?

Reply:
Maybe. I still think it is not great to make the function name so general, since it is hard-coded that this works with Choi-like MPOs (by using "Input" and "Output" explicitly in the function). However, that is only a very specific type of MPO, but the fact that it accepts all kinds of MPOs makes it seem like a much more general function. Basically what I mean is that if you had any other type of MPO that had multiple site indices per tensor, this function wouldn't be well defined.